packages/core/src/internal/motion/motion.service.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { MotionRegistry, TargetedAnimation } from './interfaces.js';
+
+const registry: MotionRegistry = {};
+
+const motionRegistry = new Proxy(registry, {
```

Comment:
Cool. I didn't know about Proxy objects.

Reply:
Yeah. I wasn't too familiar with them either. @coryrylan introduced them to the library. I like to learn new things!